Wrong version of MSV CRT linked in static Windows build

Hello - These lines set `/MD{d}` for both static and dynamic builds, but I think static should be using `/MT{d}`.

https://github.com/intel/isa-l/blob/8b7c1b80b22833e0629ccf00e62fda6b3884c7ae/Makefile.nmake#L170-L171

Trying to statically link this to library causes `warning LNK4098: defaultlib 'MSVCRT' conflicts with use of other libs`.

--- a/Makefile.nmake
+++ b/Makefile.nmake
@@ -167,8 +167,8 @@ objs = \
 INCLUDES   = -I./ -Ierasure_code/ -Iraid/ -Icrc/ -Iigzip/ -Iprograms/ -Imem/ -Iinclude/ -Itests/fuzz/ -Iexamples/ec/
 # Modern asm feature level, consider upgrading nasm/yasm before decreasing feature_level
 FEAT_FLAGS = -DHAVE_AS_KNOWS_AVX512 -DAS_FEATURE_LEVEL=10
-CFLAGS_REL = -O2 -DNDEBUG /Z7 /MD /Gy
-CFLAGS_DBG = -Od -DDEBUG /Z7 /MDd
+CFLAGS_REL = -O2 -DNDEBUG /Z7 /Gy
+CFLAGS_DBG = -Od -DDEBUG /Z7
 LINKFLAGS  = -nologo -incremental:no -debug
 CFLAGS     = $(CFLAGS_REL) -nologo -D_USE_MATH_DEFINES $(FEAT_FLAGS) $(INCLUDES) $(D)
 AFLAGS     = -f win64 $(FEAT_FLAGS) $(INCLUDES) $(D)

--- a/tools/gen_nmake.mk
+++ b/tools/gen_nmake.mk
@@ -21,8 +21,8 @@ Makefile.nmake tst.nmake: FORCE
 	@echo 'INCLUDES   = $(INCLUDE)'	>> $@
 	@echo '# Modern asm feature level, consider upgrading nasm/yasm before decreasing feature_level'	>> $@
 	@echo 'FEAT_FLAGS = -DHAVE_AS_KNOWS_AVX512 -DAS_FEATURE_LEVEL=10'	>> $@
-	@echo 'CFLAGS_REL = -O2 -DNDEBUG /Z7 /MD /Gy'		>> $@
-	@echo 'CFLAGS_DBG = -Od -DDEBUG /Z7 /MDd'			>> $@
+	@echo 'CFLAGS_REL = -O2 -DNDEBUG /Z7 /Gy'		>> $@
+	@echo 'CFLAGS_DBG = -Od -DDEBUG /Z7'			>> $@
 	@echo 'LINKFLAGS  = -nologo -incremental:no -debug'	>> $@
 	@echo 'CFLAGS     = $$(CFLAGS_REL) -nologo -D_USE_MATH_DEFINES $$(FEAT_FLAGS) $$(INCLUDES) $$(D)'	>> $@
 	@echo 'AFLAGS     = -f win64 $$(FEAT_FLAGS) $$(INCLUDES) $$(D)'	>> $@

